npm-frontend/src/Song.jsx

```diff
@@ -59,47 +75,86 @@ const RatingBox = styled.div`
   align-items: center;
 `;
 
+const SongContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const VoteContainer = styled.div`
+  position: relative;
+  right: ${props => props.removable ? "-50px" : "18px"};
+`;
+
 class Song extends Component {
   constructor(props) {
     super();
 
     this.state = {
       up: false,
       down: false,
-      votes: props.voteCount || 0
+    }
+  }
+
+  componentDidUpdate() {
+    if(this.state.up || this.state.down) {
+      this.setState({ up: false, down: false })
     }
   }
 
   upVote = () => {
-    let votes = this.state.votes
-    // If down was already clicked, need to change value by 2
-    if(this.state.down) { votes += 2; }
-    else if(!this.state.up) { votes += 1; }
-    this.setState({ up: true, down: false, votes: votes })
+    // Update vote count if up wasn't clicked
+    // If up was already clicked, don't do anything
+    if(!this.state.up) { 
+      this.setState({ up: true, down: false});
+
+      fetch(`http://localhost:8080/api/upvote?eventCode=${this.props.currEventCode}&userId=${this.props.userID}&trackId=${this.props.id}`);
```

Comment:
Didn't you change the port to 3000?

Reply:
The port for the front end is 3000. The back end is 8080 so that's where calls have to be directed to make api calls

Reply:
The thing I added to the backend was for CORS stuff cause it would block the frontend without it